### DISCUSSION TITLE
Equivalent to {incremental_where_cond} or {incremental_value} but for backfill mode ?

### DISCUSSION THREAD

**Discussion by fabrice-etanchaud** [slingdata-io/sling-cli#215] [Q&A]
Hi, sorry I have so many questions about Sling...

Like {incremental_where_cond} or {incremental_value} variables for incremental mode,
if one wants to write custom sql in backfill mode, are there any equivalent variables ?

**Comment by flarco** [comment]
It should work with `{incremental_where_cond}` or `{incremental_value}`.
Any suggestion to avoid confusion for others in the future? 🤔

**Reply by royandhika** [reply]
backfill & incremental both accept {incremental_where_cond} and {incremental_value} right?

so i can write custom sql like:
`SELECT * FROM my_schema.my_table 
WHERE {incremental_where_cond}`

`SELECT * FROM my_schema.my_table 
WHERE my_int_key > coalesce({incremental_value}, 0)  -- For numeric columns
-- or
WHERE my_timestamp > coalesce({incremental_value}, '2001-01-01')  -- For timestamp columns`

something doesnt add up, in incremental its totally understandable because the {incremental_value} = max_value from our table

but with backfill, we got 2 value range [start, end]
how do i write {incremental_value} with backfill?
